features/dd-sdk-android-trace-api/src/main/kotlin/com/datadog/android/trace/api/span/DatadogSpan.kt

```diff
@@ -64,7 +64,7 @@ interface DatadogSpan {
     /**
      * Represents the start time of the span in nanoseconds.
      */
-    val startTime: Long
+    val startTimeNano: Long
```

Comment:
just for the consistency with naming across the codebase

```suggestion
    val startTimeNanos: Long
```